### PR TITLE
Added nidorinas PPP

### PR DESCRIPTION
**Pull request by sarahcxh**

### CODE REVIEW

File: docs/team/nidorinas.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+** Documentation:
+*** Contributed to both the Developer Guide and User Guide for this project.
+*** Did cosmetic tweaks to existing contents of the User Guide: https://github.com/AY1920S2-CS2103-W14-3/main/pull/154[#154]
+** Community:
+*** PRs reviewed (with non-trivial review comments): https://github.com/AY1920S2-CS2103-W14-3/main/pull/152[#152]
+*** Reported bugs and suggestions for other teams in the class (examples:  https://github.com/AY1920S2-CS2103-T09-3/main/issues/114[1], https://github.com/AY1920S2-CS2103-T09-3/main/issues/115[2], https://github.com/AY1920S2-CS2103-T09-3/main/issues/116[3], https://github.com/AY1920S2-CS2103-T09-3/main/issues/117[4], https://github.com/AY1920S2-CS2103-T09-3/main/issues/118[5], https://github.com/AY1920S2-CS2103-T09-3/main/issues/119[6], https://github.com/AY1920S2-CS2103-T09-3/main/issues/120[7], https://github.com/AY1920S2-CS2103-T09-3/main/issues/121[8])
+** Tools:
+*** Integrated Travis with the team repo
+
+_{you can add/remove categories in the list above}_
```
Comment (Vanessa1098) [inline docs/team/nidorinas.adoc]: Should probably remove this line

File: docs/team/nidorinas.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+|_Given below are sections I contributed to the User Guide. They showcase my ability to write documentation targeting end-users._
+|===
+
+== Contributions to the Developer Guide
+
+|===
+|_Given below are sections I contributed to the Developer Guide. They showcase my ability to write technical documentation and the technical depth of my contributions to the project._
+|===
+
+== PROJECT: PowerPointLabs
```
Comment (Vanessa1098) [inline docs/team/nidorinas.adoc:53]: this line too

**Review by Vanessa1098 — COMMENTED** [review]
Looks good!

File: docs/team/nidorinas.adoc (hunk 2 above)
Comment (sarahcxh) [reply docs/team/nidorinas.adoc:53]: okk!

Comment (hhjoel) [reply docs/team/nidorinas.adoc:53]: This line actually in your PPP also!

**Pull request merged**